nucypher/crypto/keystore.py

```diff
@@ -342,15 +342,27 @@ def restore(cls, words: str, password: str, keystore_dir: Optional[Path] = None)
         return keystore
 
     @classmethod
-    def generate(cls, password: str, keystore_dir: Optional[Path] = None, interactive: bool = True) -> 'Keystore':
+    def generate(
+            cls, password: str,
+            keystore_dir: Optional[Path] = None,
+            interactive: bool = True,
+            report_mnemonic: bool = False
+            ) -> 'Keystore':
```

Comment:
Given the new optional return type this type annotation is no longer accurate.

```suggestion
            ) -> Union['Keystore', Tuple['Keystore', str]]:
```